docs/DeveloperGuide.md

```diff
@@ -236,9 +236,18 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
+* is the store manager of a milk tea shop
```

Comment:
maybe we should include the **brand** of the milk tea shop here also? as some function are only for **TGER SUGER** brand

Reply:
thanks, resolved!